webassembly.html

```diff
@@ -317,6 +317,13 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             process.</dd>
           </dl>
           <dl>
+            <dt><a href="https://www.w3.org/2008/webapps/">Web Applications Working Group</a></dt>
```

Comment:
Should we add the associated &lt;dd/> text:

> Coordinate on interoperability, performance, and ergonomics with<a href="https://www.w3.org/TR/IndexedDB/">IndexedDB</a>.

under the WebPlatform WG text?